### PR TITLE
IBX-2895 - edition labels

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-2895 |
| Versions      | all |

Add flavor labels to content to make it clear what feature goes where

### CODE REVIEW

File: docs/releases/ibexa_dxp_v3.2.md [hunk 1]
```diff
@@ -68,15 +68,15 @@ and [GraphQL](https://doc.ibexa.co/en/latest/api/graphql/#jwt-authentication).
 
 See [JWT authentication](https://doc.ibexa.co/en/latest/guide/security/#jwt-authentication) to learn how to configure this authentication method.
 
-### Searching in Ibexa Commerce with Elasticsearch
+### Searching in Ibexa Commerce with Elasticsearch [[% include 'snippets/commerce_badge.md' %]]
```
Comment (DominikaK) [inline docs/releases/ibexa_dxp_v3.2.md:71]: In v3.2 Elastic was part of https://github.com/ezsystems/ezplatform-ee/blob/v3.2.8/composer.json#L36

File: docs/tutorials/enterprise_beginner/1_get_a_starter_website.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# Step 1 — Get a starter website [[% include 'snippets/experience_badge.md' %]]
+# Step 1 — Get a starter website [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
 To set up the starter website, you need to follow these steps:
 
-## Get a clean [[= product_name_exp =]] installation
+## Get a clean [[= product_name =]] installation
 
-To begin the tutorial, you need a clean installation of [[= product_name_exp =]].
+To begin the tutorial, you need a clean installation of [[= product_name_exp =]] or [[= product_name_com =]].
```
Comment (DominikaK) [inline docs/tutorials/enterprise_beginner/1_get_a_starter_website.md:7]: Using Commerce in the tutorial will probably backfire for beginner users, due to different front page, let's not suggest this.

File: docs/releases/ibexa_dxp_v4.0.md [hunk 3]
```diff
@@ -82,20 +82,20 @@ See [Managing migrations](../api/public_php_api_managing_migrations.md) for more
...
 
 You can purge all submissions of a given form. 
 For more information, see [Forms](../guide/form_builder/forms.md#form-submission-purging).
 
 ### Hidden eCommerce features
 
-Commerce tab and all its features are now disabled by default.
-For more information, see [Enable Commerce features](../guide/config_back_office.md#enable-commerce-features).
+In [[= product_name_content =]], Commerce tab and all its features are now disabled by default.
+For more information, see [Enabling Commerce features](https://doc.ibexa.co/en/4.0/guide/config_back_office/#enable-commerce-features).
```
Comment (DominikaK) [inline docs/releases/ibexa_dxp_v4.0.md:98]: Watch out when merging to earlier branches: you're removing the section linked here in this PR.

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/guide/multisite/multisite_configuration.md [hunk 4]
```diff
@@ -164,7 +164,7 @@ ibexa:
     be sure to use SSL (https protocol).
     Otherwise, you cannot preview content in Page Builder from the Back Office.
 
-#### SiteAccess switching in Page Builder [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+#### SiteAccess switching in Page Builder
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/guide/multisite/multisite_configuration.md:167]: It is a lower level paragraph than "SiteAccesses and Page Builder", which is already marked as Experience/Commerce.

File: docs/tutorials/enterprise_beginner/1_get_a_starter_website.md (hunk 2 above)
Comment (dabrt) [inline docs/tutorials/enterprise_beginner/1_get_a_starter_website.md:7]:
```suggestion
To begin the tutorial, you need a clean installation of [[= product_name_exp =]].
```

File: docs/releases/ibexa_dxp_v3.2.md (hunk 1 above)
Comment (DominikaK) [reply docs/releases/ibexa_dxp_v3.2.md:71]: True, you're right.

File: docs/releases/ibexa_dxp_v4.0.md (hunk 3 above)
Comment (DominikaK) [reply docs/releases/ibexa_dxp_v4.0.md:98]: https://github.com/ezsystems/developer-documentation/pull/1637/files#diff-ee2c7e1cf422bdfac9c8a1035acd8fdbeecebab1dbb8aa03a292d49f2041d56bL209 removes this section, so after this is merged do 4.0, the link above will not work

File: docs/guide/customers/customer_api/configuration_for_customer_data.md [hunk 5]
```diff
@@ -3,8 +3,7 @@
 ## User Groups for private and business customers
 
 To separate business and private users in one installation, there are two User Groups in the "Shop user" group.
-You can configure those groups by using `user_group_location.business` and `user_group_location.private`
-in `config/ecommerce_parameters.yml`:
+You can configure those groups in YAML files with shop configuration, by using `user_group_location.business` and `user_group_location.private`:
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/customers/customer_api/configuration_for_customer_data.md]: I agree as a rule, but then we would have to be consistent: either mention it everywhere we refer to config files (which is a lot of places), or only in introductory topics.

**Review by DominikaK — APPROVED** [review]
Note: you can forgo merging this to 2.5, that version uses a different system of indicating product editions.

**Pull request merged**